.changeset/orange-dots-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@magicbell/project-client': patch
+---
+
+Correctly flagging nullable properties. All of them were optional before so they are likely handled correctly already and this is just affecting response validation.
```

Comment:
@smeijer do you agree that this is a `patch`?

Reply:
I do.